### PR TITLE
fix(doc): more details and fixes

### PR DESCRIPTION
**Pull request by pierre-yves-monnet**
Specify which tomcat software to download,
Precise files to copy
give more information on the properties to set

### CODE REVIEW

**Comment by tbouffard** [comment]
@pierre-yves-monnet thanks for your contribution. Can you please sign the CLA please? https://github.com/bonitasoft/bonita-doc/pull/1291#issuecomment-705899023

**Comment by pierre-yves-monnet** [comment]
Done!

File: md/bonita-as-windows-service.md [hunk 1]
```diff
@@ -9,30 +9,35 @@ For this tutorial you will need the following softwares:
 * Bonita Tomcat Bundle
 * Postgresql Database (or another database supported by Bonita)
 
-Please verify the supported version from the Support page
+Please verify the supported version in Installation/Basic Bonita Platform installation/hardware-and-software-requirements
```
Comment (educhastenier) [inline md/bonita-as-windows-service.md]: @pierre-yves-monnet please add link to the pointed page.
```suggestion
Please verify the supported version in [Installation/Basic Bonita Platform installation/hardware-and-software-requirements](hardware-and-software-requirements.md)
```

File: md/bonita-as-windows-service.md [hunk 2]
```diff
@@ -9,30 +9,35 @@ For this tutorial you will need the following softwares:
...
 
 ## Download and installation
 
 * **_Download_** apache-tomcat-x.y.exe from Tomcat website: https://tomcat.apache.org/
-* **_Install_** apache-tomcat-x.y.exe. From now on we'll call the installation path as %TOMCAT_INSTALL_FOLDER%
+
+:::warning
+download the same apache-tomcat version (visible in Installation/Basic Bonita Platform installation/hardware-and-software-requirements)
```
Comment (educhastenier) [inline md/bonita-as-windows-service.md]: @pierre-yves-monnet please add link to the pointed page.

Comment (educhastenier) [inline md/bonita-as-windows-service.md]:
```suggestion
Download the same apache-tomcat version (visible in [Installation/Basic Bonita Platform installation/hardware-and-software-requirements](hardware-and-software-requirements.md))
```

**Pull request merged**